cloud/store/raft.go

```diff
@@ -47,115 +48,75 @@ type Candidate struct {
 	NonVoter bool
 }
 
-func (f *Store) Open(isLeader bool, joiners []Candidate) (*raft.Raft, error) {
+// Open opens this store and marked as such.
+func (st *Store) Open() (err error) {
```

Comment:
Nitpick: Maybe we should move this function to `store.go` file. It seems very much related to the store struct defined in this other file.

Reply:
Nearly all function in this file are related to the Store Struct. This is  primarily because they are directly related to the RAFT layer. 
It's possible these funcs may refacotred into a different class in the future